apps/full-site-editing/full-site-editing-plugin/blocks/content-slot/index.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+function render_content_slot_block( $attributes, $content ) {
+	$align = isset( $attributes['align'] ) ? ' align' . $attributes['align'] : '';
+
+	$content = '<div class="a8c-content'. $align . '">'
+		. __( '[Renders some content]' )
```

Comment:
oh. ha. see my above comment in the static render fallback. note that while we are free to render this here, we can also send that into the editor, and even have both. but if we have the static fallback render this isn't entirely necessary.